test/benchmark/README.md

```diff
@@ -52,7 +52,7 @@ Define the following environment variables.
 > **Note:** You can also define variables locally in a script or in the command line from which the relevant command is executed.
 
 ```bash 
-    V3IO_CONF="$HOME/go/bin/v3io-custom.yaml"
+    V3IO_TSDB_CONFIG="$HOME/go/bin/v3io-custom.yaml"
```

Comment:
I think **v3io-custom.yaml** should be renamed to **v3io-tsdb-custom.yaml** or some other name that includes "tsdb" and more uniquely identifies the purpose of this file.

Reply:
You've now renamed the file to **v3io-tsdb-custom.yaml** in commit 97c69b4 (good), but there are still references to the old name:
```
$ git grep -wlic "v3io-custom\.yaml"
test/benchmark/README.md
```